README.md

```diff
@@ -153,6 +153,19 @@ default['ossec']['agent_auth']['register'] = 'yes'
 
 In other case, we just assign a different value which is not `yes`.
 
+
+## In case you're using CentOS 6 with Kibana 7.2.0
```

Comment:
Let's move this under de dependencies section, I think most of us don't completely read the README and some users might miss it.

Reply:
Hi @jm404,

Check the changes [here](https://github.com/wazuh/wazuh-chef/commit/59d4e6be6389fe57e017a6c47a66177280c3ede2)

Kind regards,

Rshad